doc/changelog.md

```diff
@@ -0,0 +1,919 @@
+# Changelog
+
+Listed here are the changes between each release of SmartSim,
+SmartRedis and SmartDashboard.
+
+## SmartSim
+
+### Development branch
+
+To be released at some future point in time
+
+Description 521, 524, 520
+
```

Comment:
I believe "521, 524, 520" should be removed.

Reply:
oop yes

Reply:
I think "Description" was still needed, just without the numbers.